managedtenants/core/addons_loader/addon.py

```diff
@@ -21,6 +21,7 @@
 # These addon IDs _MUST_ be stable and not changed or bad things will happen
 # (this applies to addon IDs in general, but it's worth pointing out here, too)
 _ADDON_OPERATOR_ADDON_IDS = ["reference-addon"]
+_PERMITTED_SUBSCRIPTION_CONFIGS = {"env"}
```

Comment:
Sets are more expensive than tuples or lists. Unless you need on-the-fly deduplication, I'd reccomend using a tuple here. Or a list, for consistency with the previous line.